payments-ui-core/api/payments-ui-core.api

```diff
@@ -5,6 +5,13 @@ public final class com/stripe/android/ui/core/BuildConfig {
 	public fun <init> ()V
 }
 
+public final class com/stripe/android/ui/core/CurrencyFormatter$Companion {
```

Comment:
Can we make it restricted so it doesn't appear in the apiDUmp

Reply:
Done